orcid-web/src/main/webapp/static/javascript/ng1Orcid/app/modules/oauthAuthorization/oauthAuthorization.component.ts

```diff
@@ -698,34 +721,26 @@ export class OauthAuthorizationComponent implements AfterViewInit, OnDestroy, On
         }
 
         //if oauth request load request info form
-        if(orcidVar.oauth2Screens || orcidVar.originalOauth2Process){
+        const urlParams = new URLSearchParams(window.location.search);
```

Comment:
Just for the records. 
Is regularly not recommended to access directly global objects like window. This restricts the app to not use optimizations like angular server-side rendering, more info: https://brianflove.com/2018/01/11/angular-window-provider/